packages/app/src/systems/Network/components/NetworksDialog/NetworkDialog.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { Box } from '@fuel-ui/react';
```

Comment:
checking PR storybook, nothing is shown in this story.

can we add a button to open dialog?

Reply:
Can you check if the storybook for `NetworksDialog` is rendering fine for you? With `viewport` parameters configured for a component, I'm getting errors locally when trying to view it's storybook.